test/e2e/next-image/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import { createNext, FileRef } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { waitFor } from 'next-test-utils'
+import path from 'path'
+import webdriver from 'next-webdriver'
+
+describe('next-image-forward-ref', () => {
+  let next: NextInstance
+
+  const appDir = path.join(__dirname, 'app')
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: new FileRef(appDir),
+      dependencies: {
+        'framer-motion': '7.6.9',
+      },
+    })
+  })
+  afterAll(() => next.destroy())
+
+  it('allows framer-motion to animate opacity', async () => {
+    const browser = await webdriver(next.url, '/framer-motion')
```

Comment:
Lets also rename the page to the same name

- `farmer-motion.js` should be `framer-motion.js`